internal/core/command/get_test.go

```diff
@@ -108,9 +118,51 @@ func TestNewGetCommandNoCorrelationIDInContext(t *testing.T) {
 
 func TestNewGetCommandInvalidBaseUrl(t *testing.T) {
 	device := testDevice
+	req := newRequestForGetWithContentType(clients.ContentTypeCBOR)
 	device.Service.Addressable.Address = "!@#$"
-	_, err := NewGetCommand(device, testCommand, "", context.Background(), nil, logger.NewMockClient())
+	_, err := NewGetCommand(device, testCommand, context.Background(), nil, logger.NewMockClient(), req)
 	if err == nil {
 		t.Errorf("The invalid URL error was not properly propagated to the caller")
 	}
 }
+
+func TestNewGetCommandContentType(t *testing.T) {
```

Comment:
Can we add negative test cases so that we ensure headers that we do not expect to be propagated are not? Also, to my other comment regarding the behavior of the function which copies headers to the proxied request, can we add a test which does not have a `Content-type` header and ensure that it is not present in the outgoing request?